doc/source/tutorial/stats/continuous_kstwo.rst

```diff
@@ -71,16 +70,16 @@ array([[-1.591,  0.2  ,  0.056,  0.056,  0.144],
 >>> Dn = np.max(Dnpm)
 >>> iminus, iplus = np.argmax(gaps, axis=0)
 >>> print('Dn- = %f (at x=%.2f)' % (Dnpm[0], x[iminus]))
-Dn- = 0.308125 (at x=0.55)
+Dn- = 0.308125 (at x=0.55)  # random
 >>> print('Dn+ = %f (at x=%.2f)' % (Dnpm[1], x[iplus]))
-Dn+ = 0.146447 (at x=-0.66)
+Dn+ = 0.146447 (at x=-0.66)  # random
 >>> print('Dn  = %f' % (Dn))
-Dn  = 0.308125
+Dn  = 0.308125  # random
 
 >>> probs = kstwo.sf(Dn, n)
 >>> print(chr(10).join(['For a sample of size %d drawn from a N(0, 1) distribution:' % n,
 ...      ' Kolmogorov-Smirnov 2-sided n=%d: Prob(Dn >= %f) = %.4f' % (n, Dn, probs)]))
-For a sample of size 5 drawn from a N(0, 1) distribution:
+For a sample of size 5 drawn from a N(0, 1) distribution:  # random
```

Comment:
This is even worse. Can't tell from where does the "random" comment come from. Can we shift this at the end of the output?

Reply:
Agreed, I will check.